lib/exceptions.json

```diff
@@ -1,15 +1,22 @@
 {
-  "webrtc": [
+  ".*": [
+    {
+      "rule": "validation.css",
+      "type": "generator.unrecognize",
+      "source": "https://www.w3.org/StyleSheets/TR/2016/base.css"
+    }
+  ],
+  "^webrtc$": [
     {
       "rule": "headers.copyright"
     }
   ],
-  "mediacapture-streams": [
+  "^mediacapture-streams$": [
```

Comment:
I thought that `-` here would be [mis]interpreted as defining a range of characters.
But I just learnt that [hyphens do not need to be escaped when they don't appear inside square brackets](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#character-sets).
Still, maybe `"^mediacapture\-streams$"` to err on the safe side, and to remind anyone who is editing the exceptions file that these metacharacters *usually* ought to be escaped&hellip;?